canary/environments/auth-with-email/package.json

```diff
@@ -3,6 +3,6 @@
   "name": "auth-with-email-environment",
   "version": "0.0.1",
   "scripts": {
-    "pull": "amplify pull --appId d38yznh7fio4c0 --envName staging"
+    "pull": "amplify pull --appId d2ek4lzondrx53 --envName dev"
```

Comment:
Oops....I'll remove this change

Reply:
Was just writing a comment about this :)